examples/message-ubench/.gitignore

```diff
@@ -0,0 +1 @@
+message-ubench
```

Comment:
None of the other examples have a `.gitignore` file.

I can see the appeal of having it, and I wouldn't mind having it on the examples, but I think there's value in consistency here, so I'd prefer to see either *all* or *none* of them having it.

Reply:
@jemc i added .gitignore for examples on the backpressure branch as well.

Reply:
If that's the road we want to go down, should we at least create a ticket to add `.gitignore`s for every example?

Reply:
I think that is reasonable jemc.